keras_cv/metrics/object_detection/mean_box_count_delta.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+import tensorflow.keras as keras
+import tensorflow.keras.initializers as initializers
+
+from keras_cv import bounding_box
+
+
+class MeanBoxCountDelta(keras.metrics.Metric):
```

Comment:
For all other Mean metrics I'm aware of (except mAP), the "Mean" prefix is omitted. Should we do that here?

Reply:
Good point - updated to omit